src/build/parse.rs

```diff
@@ -103,90 +103,58 @@ pub fn generate_asts(
                     .packages
                     .get(&module.package_name)
                     .expect("Package not found");
-                if is_dirty {
```

Comment:
Deze is weg?

Reply:
<img width="1453" alt="Screenshot 2024-04-16 at 15 38 44" src="https://github.com/teamwalnut/rewatch/assets/4375473/101ce16c-e93f-4ccf-aec6-fff3e59906b7">

Seems like a duplicate?